mobile/src/core/utils/balance/getBalanceHistoryFromTransactions.ts

```diff
@@ -10,7 +10,12 @@ const isOwnTransaction = (accountId: string, transaction: Transaction): boolean
 function getRelativeTransactionAmount (accountId: string, transaction: Transaction): number {
 
   if (isOwnTransaction(accountId, transaction)) {
-    const amountBurst = convertNQTStringToNumber(transaction.amountNQT);
+    // type 2, subtype 3 = BidOrderPlacement
+    // Todo: support Ask Order Placement?
+    // This logic is flawed/imperfect, but seems to work well enough
+    const amountBurst = transaction.type === 2 && transaction.subtype === 3 ?
```

Comment:
We have constants for that
```suggestion
    const amountBurst = transaction.type === TransactionType.Asset && transaction.subtype === TransactionAssetSubtype.BidOrderPlacement ?
```